### PR TITLE
Arxivce 1747 purge on announce

### PR DESCRIPTION
**Pull request by kyokukou**
creates a cloud function that on getting triggered that announce is complete purges the fastly cache for any papers labeled to be cleared on announcement

### CODE REVIEW

File: cloud_functions/fastly_announce_purge/cicd/env.fastly_announce_purge.arxiv-development.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+ENVIROMENT: DEVLOPMENT
```
Comment (bmaltzan) [inline cloud_functions/fastly_announce_purge/cicd/env.fastly_announce_purge.arxiv-development.yaml]: typo?

**Review by bmaltzan — APPROVED** [review]

File: cloud_functions/fastly_announce_purge/cicd/env.fastly_announce_purge.arxiv-production.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+ENVIROMENT: PRODUCTION
```
Comment (bmaltzan) [inline cloud_functions/fastly_announce_purge/cicd/env.fastly_announce_purge.arxiv-production.yaml]: typo?

File: cloud_functions/fastly_announce_purge/cicd/env.fastly_announce_purge.arxiv-development.yaml (hunk 1 above)
Comment (kyokukou) [reply cloud_functions/fastly_announce_purge/cicd/env.fastly_announce_purge.arxiv-development.yaml]: well, TIL....

Comment (norbusan) [reply cloud_functions/fastly_announce_purge/cicd/env.fastly_announce_purge.arxiv-development.yaml]: 2 typos?

File: cloud_functions/fastly_announce_purge/cicd/env.fastly_announce_purge.arxiv-development.yaml [hunk 3]
```diff
@@ -1 +1 @@
-ENVIROMENT: DEVLOPMENT
\ No newline at end of file
+ENVIRONMENT: DEVLOPMENT
```
Comment (norbusan) [inline cloud_functions/fastly_announce_purge/cicd/env.fastly_announce_purge.arxiv-development.yaml]: DEVELOPMENT?

**Pull request merged**